Playground/RxSwiftExtPlayground.playground/Pages/UIScrollView.reachedBottom.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,52 @@
+/*:
+ > # IMPORTANT: To use `RxSwiftExtPlayground.playground`, please:
+
+ 1. Make sure you have [Carthage](https://github.com/Carthage/Carthage) installed
+ 1. Fetch Carthage dependencies from shell: `carthage bootstrap --platform ios`
+ 1. Build scheme `RxSwiftExtPlayground` scheme for a simulator target
+ 1. Choose `View > Show Debug Area`
+ */
+
+//: [Previous](@previous)
+
+import RxSwift
+import RxCocoa
+import RxSwiftExt
+import PlaygroundSupport
+import UIKit
+
+/*:
+ ## reachedBottom
+
+ The `reachedBottom` operator provides a sequence that emits every time the `UIScrollView` is scrolled to the bottom.
```

Comment:
Also just realised reachedBottom isn't an operator...

Reply:
Lol, yes. Can i open additional Pr on that & readme.md?

Reply:
Just pushed these on master. Let me know if you have any notes for it.

Reply:
Looks perfect, thank your for comments